bundle.yml

```diff
@@ -61,7 +61,7 @@ projects :
 
     - atlas :
         git     : https://github.com/ecmwf/atlas
-        version : 0.34.0
+        version : feature/MultiField
```

Comment:
Just out of curiosity: Is there a timeline for including this feature in a release?

Reply:
Willem is going to include the feature in the release very soon.